src/styles/index.scss

```diff
@@ -1,3 +1,65 @@
+@import '../utils/variables';
+
 body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
   margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  box-sizing: border-box;
+  flex-wrap: wrap;
+  width: $cal-width;
+  gap: $day-gap;
+  padding: $cal-padding;
+
+  &__day {
+    box-sizing: border-box;
```

Comment:
Since 'box-sizing: border-box;' is recommended to be applied globally, this line can be removed if the global style is set.